src/srcinfo.sh

```diff
@@ -282,3 +277,53 @@ function srcinfo.print_var() {
         unset "${e%\[*}"
     done
 }
+
+# @description Output a specific variable from .SRCINFO
+#
+# @example
+#   srcinfo.match_pkg .SRCINFO pkgbase
+#   srcinfo.match_pkg .SRCINFO pkgdesc $(srcinfo.match_pkg .SRCINFO pkgbase)
+#
+# @arg $1 string .SRCINFO file path
+# @arg $2 string Variable or Array to search
+# @arg $3 string Package name or base to get output for
+function srcinfo.match_pkg() {
+    local declares d bases b guy match out srcfile="${1}" search="${2}" pkg="${3}"
```

Comment:
Add the `:?message` part to the positional variables if they are required.

Reply:
No because then it’s going to print something out and not work for -z tests.